Src/Campus.Master.API/Controllers/TaskController.cs

```diff
@@ -70,15 +70,15 @@ public TaskController(ITaskService taskService, ILogger<TaskController> logger)
                     Description = "Description",
                     Priority = "Priority",
                     Tag = "Tag",
-                    Deadline = "00-00-0000"
+                    Deadline = DateTime.MinValue
```

Comment:
Implement **Task** controller

Reply:
We need to define type of filter that can be applied

Reply:
We can leave filtering endpoint empty but implement other